### PR TITLE
HomeWork 3&4

### PR DESCRIPTION
**Pull request by KseniaKotova**
Task: https://www.figma.com/file/D3hs2PC8uClFPe6iTTNXat/TMS_front?node-id=617%3A1280 header&banner
Created:

### CODE REVIEW

**Review by Stalexus — APPROVED** [review]

File: lesson 3/style.css [hunk 1]
```diff
@@ -0,0 +1,189 @@
+h1, p{
+    margin:0;
+}
+body{
+    margin: 0 auto;
+    font-family: 'Open Sans';
```
Comment (shuliakleonid) [inline lesson 3/style.css]: add family

**Review by shuliakleonid — APPROVED** [review]

File: lesson 3/style.css (hunk 1 above)
Comment (KseniaKotova) [reply lesson 3/style.css]: Fixed

**Review by shuliakleonid — APPROVED** [review]

**Comment by AlexisKozlov** [comment]
Check your delpoy

File: lesson 3/style.css [hunk 2]
```diff
@@ -13,26 +13,21 @@ button{
 .header{
     display: flex;
     justify-content: space-between; 
-    background-color: #000000;
+    background-color: #000000; 
     width: 1510px;
```
Comment (shuliakleonid) [inline lesson 3/style.css]: #000 it correct too

File: lesson 3/style.css [hunk 3]
```diff
@@ -187,3 +180,45 @@ button{
 .banner__wrapper{
     display: flex;
 }
+
+.social__line{
+    margin: 0 auto;
+    width: 1510px;
+    /* height: 140px; */
```
Comment (shuliakleonid) [inline lesson 3/style.css]: delete comment if you don't need it

**Review by shuliakleonid — APPROVED** [review]

File: lesson 3/style.css [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+    width: 1510px;
+    display: flex;
+    background-color: #e5e5e5;
+}
+
+.tab__table{
+    /* margin-top: 99px;
+    margin-left: 170px;
+    margin-bottom: 50px; */
+    margin: 99px 0px 50px 170px;
```
Comment (shuliakleonid) [inline lesson 3/style.css]: comments delete

**Review by shuliakleonid — APPROVED** [review]

**Comment by shuliakleonid** [comment]
## Good

**Review by AlexisKozlov — APPROVED** [review]
Good, but i want see deploy =)

**Pull request merged**